wgpu/examples/mipmap/main.rs

```diff
@@ -6,7 +6,7 @@ use std::{borrow::Cow, f32::consts, mem, num::NonZeroU32};
 use wgpu::util::DeviceExt;
 
 const TEXTURE_FORMAT: wgpu::TextureFormat = wgpu::TextureFormat::Rgba8UnormSrgb;
-const MIP_LEVEL_COUNT: u32 = 9;
```

Comment:
So that the base level texture has the same resolution as before (just adding an extra mip pass to cover the final level), otherwise it probably would affect the test results